_posts/2015-12-28-twis-45.md

```diff
@@ -0,0 +1,30 @@
+---
+layout:     post
+title:      This Weeks In Servo 45
+date:       2015-12-28 20:30:00
+summary:    What's up with Servo for the week of 21 Dec 2015 - 28 Dec 2015
+categories:
+---
+
+In the [last week](https://github.com/pulls?page=1&q=is%3Apr+is%3Amerged+closed%3A2015-12-21..2015-12-28+user%3Aservo), we landed 25 PRs in the Servo organization's repositories. Given the Christmas holiday, it's not too surprising that the number is a bit lower than usual!
+
+This week brings us some of the first progress on Stylo, which is an effort to make it possible to use Servo's style system within Firefox/Gecko.
+
+
+###Notable Additions
+
+- ecoal [added](https://github.com/servo/saltfs/pull/182) support for running Servo on Linux over EGL instead of GLX
```

Comment:
I assume you mean that the https://github.com/servo/rust-layers/pull/220 work hasn't landed and that this commit was only for WebGL stuff?